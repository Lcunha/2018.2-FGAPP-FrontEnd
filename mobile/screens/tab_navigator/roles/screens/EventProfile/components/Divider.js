import React, { Component } from "react"
import { View} from "react-native"

export default class Divider extends Component {
	render() {
		const widthSize = this.props.size != null ? this.props.size : 100
		return (
			<View
				style={{
					borderWidth: 0.25,
					width: widthSize,
					opacity: 0.25,
					margin: 20,
					alignSelf: "center"
				}}
			/>
		)
	}
}
