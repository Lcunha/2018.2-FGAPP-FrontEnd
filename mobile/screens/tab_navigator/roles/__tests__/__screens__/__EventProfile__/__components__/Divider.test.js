import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import {shallow} from 'enzyme';
import Enzyme from 'enzyme';
import renderer from 'react-test-renderer';
import Divider from '../../../../screens/EventProfile/components/Divider';

Enzyme.configure({adapter: new Adapter()});

it('renders correctly', () => {
  const tree = renderer.create(<Divider />).toJSON();
  expect(tree).toMatchSnapshot();
});
