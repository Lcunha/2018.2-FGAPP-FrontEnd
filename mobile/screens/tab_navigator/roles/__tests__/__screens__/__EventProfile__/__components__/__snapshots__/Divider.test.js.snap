// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "borderWidth": 0.25,
      "margin": 20,
      "opacity": 0.25,
      "width": 100,
    }
  }
/>
`;
